tests/test_cookie_signing.py

```diff
@@ -64,6 +64,13 @@ def test_set_and_get_cookie_with_existing_cookie():
     assert REQUEST.get_cookie('test') == 'testvalue'
 
 
-def test_set_and_get_cookie_http_only():
-    REQUEST.cookie('test', 'testvalue', http_only=True)
+def test_set_and_get_cookie_with_http_only():
+    REQUEST.cookie('test', 'testvalue')
+    assert REQUEST.get_cookie('test') == 'testvalue'
+    assert 'HttpOnly' in REQUEST.cookies[0][1]
+
+
+def test_set_and_get_cookie_without_http_only():
```

Comment:
Ok so just change this test here to this:

```python
def test_set_and_get_cookie_without_http_only():
    REQUEST.cookies = []
    REQUEST.cookie('test', 'testvalue', http_only=False, encrypt=False)
    assert REQUEST.get_cookie('test', decrypt=False) == 'testvalue'
    assert 'test=testvalue; Path=/' in REQUEST.cookies[0][1]
```

Reply:
Ready, added the these test and request class in the method cookie changed the declaration like this: 

def cookie(self, key, value, encrypt=True,
               http_only="HttpOnly;", path='/', expires=''):

Because otherwise the test failed.